_posts/blog/2016-09-06-hack-week.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "upday Hack Week"
+description: Three days, 49 participants, 30+ ideas.
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Three days, 49 participants and 30+ ideas. Find out what we did during our Hack-Week, why and what did we learn from it.
+tags: [Development, Hackathon, Hack-Week]
+date: 2016-09-06T15:39:55-04:00
+---
+
+After two smaller hackathons: one for Android developers, dedicated to testing; and the other one for the entire engineering, targeted to improving our continuous integration system, we decided that it's time to go to the next level. We wanted to expand the innovation opportunity outside the engineering teams so we created our own version of a Hack-Week.
+
+Let's discover together who participated in this, how were the ideas pitched, who worked on implementing them and most of all what we learned from all of this.
+
+## Organization
+
+### Who
+
+Developers, product owners, editors, finance, business intelligence or marketing colleagues, all have one thing in common: everyone has lots of ideas! We wanted to give each and every person a chance to see their ideas come true. We know that there's always space for improvement on all levels, therefore we opened the Hack-Week to non-technicals also.
```

Comment:
Call "non-technicals" something that focuses on what they *can* do, rather than what they can't. The Hack-Week gave the opportunity for close collaboration between people who usually have a more distant working relationship. The closeness afforded by the Hack-Week meant the we didn't have those same boundaries.